platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/binding/api/impl/ValidatorImpl.java

```diff
@@ -62,28 +62,28 @@ public void validateAuthnRequest(
       String signature,
       boolean strictSignature)
       throws SimpleSign.SignatureException, ValidationException {
+
     if (strictSignature
         && authnRequest.getAssertionConsumerServiceURL() != null
         && (authnRequest.getSignature() == null && signature == null)) {
       throw new IllegalArgumentException(
           "Invalid AuthnRequest, defined an AssertionConsumerServiceURL, but contained no identifying signature.");
     }
 
-    if (authnRequest.getRequestedAuthnContext() != null) {
-      if (authnRequest.isPassive()
-          && authnRequest
-              .getRequestedAuthnContext()
-              .getComparison()
-              .equals(AuthnContextComparisonTypeEnumeration.EXACT)
-          && authnRequest
-              .getRequestedAuthnContext()
-              .getAuthnContextClassRefs()
-              .stream()
-              .map(AuthnContextClassRef::getAuthnContextClassRef)
-              .anyMatch(PKI_SAML_CONTEXTS::contains)) {
-        throw new IllegalArgumentException(
-            "Unable to passively log user in when not specifying PKI AuthnContextClassRef");
-      }
+    if (authnRequest.getRequestedAuthnContext() != null
```

Comment:
Combines the nested `if` with the outer `if`.

Reply:
I don't think they are equivalent. There should be parenthesis around `isPassive` and the anded condition immediately after it.

Reply:
Good catch. I believe these are equivalent but it is worth more looking into. Will comment again after more analysis.

Reply:
These are equivalent.